### PR TITLE
added new css handle on filter navigator component inside of search-result

### PR DESCRIPTION
**Pull request by felipeireslan**
#### What is the purpose of this pull request?
add a new css class on filter-navigator component

#### What problem is this solving?
now it's possible to style this element with native css handle and without deprecated css selectors

#### How should this be manually tested?
Go to any VTEX IO store, make a search for products and Inspect Filter Navigator component with DevTools, you will could see the element below namespace `vtex-search-result-3-x-filters` without any css classes

#### Screenshots or example usage
Current element without css handle -> https://prnt.sc/s843za
Css Handle added -> https://prnt.sc/s84331

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/searchResult.css [hunk 1]
```diff
@@ -56,6 +56,9 @@
   grid-row: filters;
 }
 
+.filtersWrapper {
+}
+
```
Comment (Klynger) [inline react/searchResult.css:61]: No need for this change too
```suggestion
```

File: react/FilterNavigator.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ import styles from './searchResult.css'
 import { CATEGORIES_TITLE } from './utils/getFilters'
 import { newFacetPathName } from './utils/slug'
 
-const CSS_HANDLES = ['filter__container', 'filterMessage']
+const CSS_HANDLES = ['filter__container', 'filterMessage', 'filtersWrapper']
```
Comment (Klynger) [inline react/FilterNavigator.js:26]: Could you add this change too?

```suggestion
const CSS_HANDLES = ['filter__container', 'filterMessage', 'filtersWrapper'] as const
```

**Comment by felipeireslan** [comment]
@Klynger suggestions done. Thankyou for you support!

File: manifest.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "search-result",
   "vendor": "vtex",
-  "version": "3.56.1",
```
Comment (Klynger) [inline manifest.json:4]: Oh, sorry, I suggested that you remove this, but you just have to keep it as it was before

Comment (felipeireslan) [reply manifest.json]: My bad, updated!

File: react/FilterNavigator.js [hunk 4]
```diff
@@ -137,7 +137,7 @@ const FilterNavigator = ({
         </div>
       ) : (
         <Fragment>
-          <div className={filterClasses}>
+          <div className={handles.filtersWrapper}>
```
Comment (klzns) [inline react/FilterNavigator.js:140]: The previous classes should be kept in place:

```suggestion
          <div className={`${filterClasses} ${handles.filtersWrapper}`}>
```

**Comment by klzns** [comment]
@allcontributors add @felipeireslan for code

**Comment by klzns** [comment]
@felipeireslan thanks! 🚀

**Pull request merged**